CMakeLists.txt

```
@@ -18,6 +18,9 @@ if (NOT ASSIMP_FOUND)
   # find *absolute* paths to ASSIMP_LIBRARIES
   set(ASSIMP_INCLUDE_DIRS ${ASSIMP_PC_INCLUDE_DIRS})
   find_library(ASSIMP_LIBRARIES assimp HINTS ${ASSIMP_LIBRARY_DIRS})
```

Please try this instead:
```suggestion
   find_library(ASSIMP_LIBRARIES assimp HINTS ${ASSIMP_PC_LIBRARY_DIRS})
```